modules/account-lib/test/unit/coin/algo/transactionBuilder/keyRegistrationBuilder.ts

```diff
@@ -107,6 +107,31 @@ describe('Algo KeyRegistration Builder', () => {
       should.deepEqual(txJson.genesisHash.toString('base64'), genesisHash);
     });
 
+    it('should build a offline key registration transaction without non participation', async () => {
+      builder
+        .sender({ address: sender.address })
+        .fee({ fee: '1000' })
+        .firstRound(1)
+        .lastRound(100)
+        .testnet()
+        .numberOfSigners(1)
+        .nonParticipation(false);
```

Comment:
is necessary to add this? is it not the default value?

Reply:
Yes, but I want to be specific... Just to be clear.